app/modules/cart/cart-link/CartLink.tsx

```diff
@@ -0,0 +1,25 @@
+'use client'
+
+import Link from 'next/link'
+import cl from './CartLink.module.scss'
+import names from 'classnames'
+import { routes } from '@/app/modules/core/routes'
+import useFetchCart from '@/app/services/react-query/hooks/useFetchCart'
+
+const CartLink = () => {
+  const cart = useFetchCart()
+
+  return (
+    cart.data && (
```

Comment:
I think we need to show cart icon always, not only when we have some data, the only thing that should change is the number of items in card.